R/topics.R

```diff
@@ -167,3 +167,56 @@ topic_must <- function(message, topic) {
     x = paste0("Not ", encodeString(topic, quote = "'"))
   ))
 }
+
+content_info <- function(content_entry, index, pkg, section) {
+
+  if (grepl(".*::.*", content_entry)) {
+    names <- strsplit(content_entry, "::")[[1]]
+    if (!rlang::is_installed(names[1])) {
+      abort(
+        sprintf(
+          "%s must be installed if it is mentioned in the reference index.",
+          names[1]
+        )
+      )
+    }
+
+    path <- downlit::href_topic(names[2], names[1])
+    if (is.na(path)) {
+      abort(
+        sprintf(
+          "Could not find an href for topic %s of package %s",
+          names[2], names[1]
+        )
+      )
+    }
+
+    rd_name <- paste0(downlit:::find_rdname(names[1], names[2]), ".Rd")
```

Comment:
@hadley not sure how else I could find the title of the topic

Reply:
alternatively, of course, we could let users add text.

Reply:
letting users add text might be a good thing for https://github.com/r-lib/rlang/issues/959#issuecomment-642730853 as well